array/zfp/pointer1.h

```diff
@@ -21,10 +22,18 @@ class pointer {
   friend class array1;
   friend class reference;
   explicit pointer(reference r) : ref(r) {}
+public:
```

Comment:
Why should this ctor be made public?  It is available from the array being passed in, i.e., &array[i].

Reply:
In Kripke, I use `baseptr = typename type::pointer(store, 0);`. I can replace it by `baseptr = &(store[0]);`.